src/neighbor.cpp

```diff
@@ -2976,6 +2976,16 @@ bigint Neighbor::get_nneigh_half()
   return nneighhalf;
 }
 
+double **Neighbor::get_xhold()
+{
+  // Returns the pointer containing the last positions stored by the NL builder,
```

Comment:
please move this comment to be before the new method, and include the separator like the other methods have
/* -------
    comment here
--------- */